modules/ROOT/pages/accessibility.adoc

```diff
@@ -0,0 +1,16 @@
+= Accessibility
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+All Anypoint API Community Manager components, including both components provided by MuleSoft and components in Salesforce Experience Cloud, comply with the Web Content Accessibility Guidelines (WCAG) 2.1 AA level accessibility standard, and enable the creation of sites compliant with this standard and level.
+
+Because MuleSoft customers design and brand their API communities, accessibility assessment for sites built with API Community Manager depends on the customer's choice of design, structure, look and feel, visual, user interface, and behavior elements. Customers are responsible for accessibility assessment of these aspects on their specific community.
```

Comment:
"in" a community rather than "on"

Reply:
Also, take a look at the list of dependencies: for example, is it redundant to mention "look and feel" and then also "visual" and "user interface"?

Reply:
Agreed, better to trim that list. Done. Also changed "on" to "in."